primitives/runtime/src/lib.rs

```diff
@@ -535,6 +535,10 @@ pub type DispatchOutcome = Result<(), DispatchError>;
 /// - The extrinsic supplied a bad signature. This transaction won't become valid ever.
 pub type ApplyExtrinsicResult = Result<DispatchOutcome, transaction_validity::TransactionValidityError>;
 
+/// Same as `ApplyExtrinsicResult` but augmented with `PostDispatchInfo` on success.
+pub type ApplyExtrinsicResultWithInfo<T> =
```

Comment:
(unrelated to this PR) might be worth making a proposal issue later to rename these into something shorter. 

For example I think `Apply Extrinsic` is too verbose, we can name it just `Apply` or `Dispatch`

Reply:
Yes. It should be a separate commit that renames all of those `Info` stuff at once. It is inconsistent at the moment.